### PR TITLE
Add iterativeWGCNA package

### PR DESCRIPTION
**Pull request by thompsonsed**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/iterativewgcna/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+test:
+  imports:
+    - iterativeWGCNA
+    - iterativeWGCNA.io
+    - iterativeWGCNA.r
+
+about:
+  home: "http://github.com/cstoeckert/iterativeWGCNA"
+  license: "GNU"
+  license_family: "OTHER"
```
Comment (marcelotrevisani) [inline recipes/iterativewgcna/meta.yaml]:
```suggestion
  license_family: "BSD"
```

File: recipes/iterativewgcna/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+test:
+  imports:
+    - iterativeWGCNA
+    - iterativeWGCNA.io
+    - iterativeWGCNA.r
+
+about:
+  home: "http://github.com/cstoeckert/iterativeWGCNA"
+  license: "GNU"
```
Comment (marcelotrevisani) [inline recipes/iterativewgcna/meta.yaml]:
```suggestion
  license: "BSD-3-Clause"
```

File: recipes/iterativewgcna/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  license_family: "OTHER"
+  license_file: "LICENSE.txt"
+  summary: "Iterative application of WGCNA"
+  doc_url: "https://www.biorxiv.org/content/10.1101/234062v1"
+  dev_url: ""
```
Comment (marcelotrevisani) [inline recipes/iterativewgcna/meta.yaml]:
```suggestion
```

File: recipes/iterativewgcna/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "1ee490a1b7a0e1164fda08d38053a17f974f82f0237b8c2864bd12a28f3a8efb"
+
+build:
+  number: 0
```
Comment (marcelotrevisani) [inline recipes/iterativewgcna/meta.yaml]: Can it be ``noarch``?
```suggestion
  number: 0
  noarch: python
```

File: recipes/iterativewgcna/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,44 @@
+{% set name = "iterativeWGCNA" %}
+{% set version = "1.1.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "1ee490a1b7a0e1164fda08d38053a17f974f82f0237b8c2864bd12a28f3a8efb"
```
Comment (marcelotrevisani) [inline recipes/iterativewgcna/meta.yaml]: Please remove the wuotes from ``sha256``
```suggestion
  sha256: 1ee490a1b7a0e1164fda08d38053a17f974f82f0237b8c2864bd12a28f3a8efb
```

File: recipes/iterativewgcna/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (marcelotrevisani) [inline recipes/iterativewgcna/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipes/iterativewgcna/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+requirements:
+  host:
```
Comment (marcelotrevisani) [inline recipes/iterativewgcna/meta.yaml]: In ``host`` I think you just need ``python`` and ``pip``

Comment (thompsonsed) [reply recipes/iterativewgcna/meta.yaml]: I think you're correct, thanks.

File: recipes/iterativewgcna/LICENSE.txt [hunk 8]
```diff
@@ -0,0 +1,13 @@
+BSD 3-clause license
+Copyright (c) 2015-2019, conda-forge
```
Comment (chrisburr) [inline recipes/iterativewgcna/LICENSE.txt]: This is the wrong licence. It should be the one of the upstream package: https://github.com/cstoeckert/iterativeWGCNA/blob/master/LICENSE

The `licence` and `licence_family` fields should then be `GPL-2.0-only` and `GPL` respectively.

Comment (thompsonsed) [reply recipes/iterativewgcna/LICENSE.txt]: Apologies, now corrected.

File: recipes/iterativewgcna/meta.yaml [hunk 9]
```diff
@@ -21,7 +21,8 @@ requirements:
   run:
     - matplotlib
     - python
-    - rpy2
+    - rpy2  # [not win]
+    - rpy2 <3.0  # [win]
```
Comment (chrisburr) [inline recipes/iterativewgcna/meta.yaml]: Sorry it's not immediately obvious to me. Why is this needed?

Comment (thompsonsed) [reply recipes/iterativewgcna/meta.yaml]: From [this thread](https://bitbucket.org/rpy2/rpy2/issues/577/windows-conda-rpy2-version) and my own experience, rpy2 >= 3.0 doesn't support Windows fully yet. Hopefully this will be fixed in the future.
